src/test/java/org/primeframework/mvc/RoutePrinterTest.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2024, Inversoft Inc., All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ */
+package org.primeframework.mvc;
+
+import com.google.inject.Inject;
+import org.testng.annotations.Test;
+
+/**
+ * Example of using the {@link RoutePrinter utility}
+ */
+@Test
+public class RoutePrinterTest extends PrimeBaseTest {
```

Comment:
This test doesn't assert anything and will just print a bunch of stuff to the console. Do we need it, and if so, should we disable it by default? 

If we keep it, it seems like you should at least assert something to prove that changing the variables actually does something.

Reply:
it's more just an example of an easy way to 'use' the RoutePrinter class. it could also go in documentation or what not.